org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/WollokLauncherParameters.xtend

```diff
@@ -55,113 +56,115 @@ class WollokLauncherParameters {
 		buildListOption(sb, libraries, "lib", ',')
 		buildListOption(sb, wollokFiles, "wf", ' ')
 		sb.toString
 	}
-	
+
 	def appendIfNotNull(StringBuilder builder, Object parameterValue, String parameterKey) {
 		if (parameterValue !== null) builder.append("-" + parameterKey + " " + parameterValue.toString).append(" ")
 	}
 
-	def buildNumberPreferences(StringBuilder sb){
-		if(numberOfDecimals !== null) sb.append("-numberOfDecimals ").append(numberOfDecimals).append(" ")
-		if(printingStrategy !== null) sb.append("-printingStrategy ").append(printingStrategy).append(" ")
-		if(coercingStrategy !== null) sb.append("-coercingStrategy ").append(coercingStrategy).append(" ")
+	def buildNumberPreferences(StringBuilder sb) {
+		if (numberOfDecimals !== null) sb.append("-numberOfDecimals ").append(numberOfDecimals).append(" ")
+		if (printingStrategy !== null) sb.append("-printingStrategy ").append(printingStrategy).append(" ")
+		if (coercingStrategy !== null) sb.append("-coercingStrategy ").append(coercingStrategy).append(" ")
 	}
-	
+
 	def buildListOption(StringBuilder sb, List<String> options, String option, char separator) {
-		if(!options.empty) {
+		if (!options.empty) {
 			sb.append("-").append(option).append(" ")
-			for(var i = 0 ; i < options.length; i++) {
-				sb.append(options.get(i)).append( if (i < options.size() - 1) separator else " ")
+			for (var i = 0; i < options.length; i++) {
+				sb.append(options.get(i)).append(if(i < options.size() - 1) separator else " ")
 			}
-		}	
+		}
 	}
 
 	def parse(String[] args) {
+		args.forEach[println(it)]
 		val parser = new OptionalGnuParser
 		val cmdLine = parser.parse(options, args, false)
-		
+
 		hasRepl = cmdLine.hasOption("r")
 		dynamicDiagramActivated = cmdLine.hasOption("dd")
-		
+
 		tests = cmdLine.hasOption("t")
 		testPort = parseParameterInt(cmdLine, "testPort")
 		dynamicDiagramPort = parseParameterInt(cmdLine, "dynamicDiagramPort")
-		
+
 		severalFiles = cmdLine.hasOption("severalFiles")
 		folder = parseParameterString(cmdLine, "folder")
-		
 		jsonOutput = cmdLine.hasOption("jsonOutput")
-		
+
 		noAnsiFormat = cmdLine.hasOption("noAnsiFormat")
 
 		requestsPort = parseParameterInt(cmdLine, "requestsPort")
 		eventsPort = parseParameterInt(cmdLine, "eventsPort")
-		
+
+		validate = !cmdLine.hasOption("dontValidate")
 		numberOfDecimals = parseParameterInt(cmdLine, "numberOfDecimals", null)
 		printingStrategy = parseParameterString(cmdLine, "printingStrategy")
 		coercingStrategy = parseParameterString(cmdLine, "coercingStrategy")
-		
+
 		if ((requestsPort == 0 && eventsPort != 0) || (requestsPort != 0 && eventsPort == 0)) {
 			throw new RuntimeException(Messages.WollokLauncher_REQUEST_PORT_EVENTS_PORT_ARE_BOTH_REQUIRED)
 		}
-		
+
 		parseLibraries(cmdLine)
 		parseWollokFiles(cmdLine)
-		
-		if (!wollokFiles.empty && hasRepl && !wollokFiles.get(0).endsWith(".wlk")){
+
+		if (!wollokFiles.empty && hasRepl && !wollokFiles.get(0).endsWith(".wlk")) {
```

Comment:
I saw last validation several times, we should add an extension method to validate Wollok file extension once (not this time, for a future occasion)